iroh-bytes/src/store/file.rs

```diff
@@ -1153,12 +1155,21 @@ impl StoreInner {
     fn temp_file_name(&self) -> PathBuf {
         self.path_options.temp_file_name()
     }
+
+    async fn shutdown(&self) {
```

Comment:
God how I hate all this boilerplate. But no macros please. At some point I will write an actor framework to solve this...